vm/rust/src/juno_state_reader.rs

```diff
@@ -167,22 +171,31 @@ impl StateReader for JunoStateReader {
     }
 
     /// Returns the compiled class hash of the given class hash.
-    fn get_compiled_class_hash(
+    fn get_compiled_class_hash(&self, _class_hash: ClassHash) -> StateResult<CompiledClassHash> {
+        unimplemented!()
+    }
+}
+
+impl UpdatableState for JunoStateReader {
+    fn apply_writes(
         &mut self,
-        _class_hash: ClassHash,
-    ) -> StateResult<CompiledClassHash> {
+        writes: &StateMaps,
+        class_hash_to_class: &ContractClassMapping,
+        visited_pcs: &HashMap<ClassHash, HashSet<usize>>,
+    ) {
         unimplemented!()
     }
 }
```

Comment:
Why do we need this function?

Reply:
it's part of `UpdatableState` interface without it we won't compile rust